### ISSUE TITLE
Fails to build on 0.6

### ISSUE THREAD

**Issue by nbecker** [JuliaDSP/DSP.jl#148]
Tried building on 
julia-0.6.0-0.20170131.fc25.x86_64

```
INFO: Precompiling module DSP.
ERROR: LoadError: LoadError: TypeError: Type{...} expression: expected UnionAll, got Base.#FloatRange
Stacktrace:
 [1] anonymous at ./<missing>:2
while loading /home/nbecker/.julia/v0.6/DSP/src/periodograms.jl, in expression starting on line 352
while loading /home/nbecker/.julia/v0.6/DSP/src/DSP.jl, in expression starting on line 7
ERROR: LoadError: Failed to precompile DSP to /home/nbecker/.julia/lib/v0.6/DSP.ji.
Stacktrace:
 [1] eval_user_input(::Any, ::Base.REPL.REPLBackend) at ./REPL.jl:66
 [2] macro expansion at ./REPL.jl:97 [inlined]
 [3] (::Base.REPL.##1#2{Base.REPL.REPLBackend})() at ./event.jl:73
while loading /home/nbecker/julia-test/bench1.jl, in expression starting on line 11
```

**Comment by bjarthur** [comment]
see https://github.com/JuliaDSP/DSP.jl/pull/147

**Comment by simonster** [comment]
Should be fixed by #147
